plutus-playground-client/src/MainFrame.purs

```diff
@@ -357,9 +357,6 @@ eval (RemoveSimulationSlot index next) = do
 
 eval (ModifyWallets action next) = do
   modifying (_simulations <<< _current <<< _wallets) (evalWalletEvent action)
-  case action of
-    (RemoveWallet _) -> assign (_simulations <<< _current <<< _actions) []
-    _ -> pure unit
```

Comment:
I don't quite understand, what was the previous behaviour when you delete a wallet, to reset the whole simulation?

Reply:
Yes. 😊